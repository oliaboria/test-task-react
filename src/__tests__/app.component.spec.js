import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import React from 'react';

import App from '../app.component';

describe('App component', () => {
    it('should render app', () => {
        const wrapper = shallow(<App />);
        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });
});
