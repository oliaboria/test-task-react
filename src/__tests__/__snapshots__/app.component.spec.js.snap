// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render app 1`] = `
<div
  className="app-container"
>
  <AppRouter />
</div>
`;
